feathr_project/feathr/utils/job_utils.py

```diff
@@ -74,4 +74,14 @@ def get_result_df(client: FeathrClient, format: str = None, res_url: str = None,
     
     if local_folder is None:
         tmp_dir.cleanup()
-    return result_df
\ No newline at end of file
+    return result_df
+
+def copy_files(client: FeathrClient, source_url: str, target_url: str = None):
+    source_url: str = source_url or client.get_job_result_uri(block=True, timeout_sec=1200)
+    if source_url is None or target_url is None:
+        raise RuntimeError("source_url None. Please make sure either you provide a source_url or make sure the job finished in FeathrClient has a valid result URI.")
```

Comment:
One of source_url or target_url is None. Please make sure either you provide a source_url or make sure the job finished in FeathrClient has a valid result URI.

Reply:
Checked them separately.